CHANGELOG.md

```diff
@@ -28,6 +28,8 @@ Main (unreleased)
   `prometheus.operator` component which discovered no custom resources.
   (@rfratto)
 
+- Fix issue with nil tls config reference when using oauth2. (@mattdurham) 
```

Comment:
What is the "issue?" 

Also, I wouldn't mention `nil` since that's a Go specific term that isn't exposed to users, unlike a panic would be.

```suggestion
- Fix panic when using `oauth2` without specifying `tls_config`. (@mattdurham) 
```